README.md

```diff
@@ -39,55 +73,206 @@ tools (such as `ceph-installer` and `gdeploy`):
+`/usr/share/ansible/roles/tendrl-ansible.tendrl-server`.
+Each role has it's own `README.md` file, where you can find all details about
+it's usage.
+
+Playbooks are available in `/usr/share/doc/tendrl-ansible-1.6.0/` directory,
+where `1.6.0` is version of tendrl-ansible package.
+
+
+## What should I know before using tendrl-ansible?
+
+You need to know [how to use
+ansbile](https://docs.ansible.com/ansible/latest/intro.html) and how to deploy
+and use ssh public keys.
+
+Moreover since this README file can't provide all details about Tendrl, you
+should read [Tendrl installation
+documentation](https://github.com/Tendrl/documentation/wiki/Tendrl-release-latest)
+as well.
+
+And last but not least, both `tendrl-server` and `tendrl-storage-node` contains
+many variables which one can use to tweak the installation. See README files of
+the roles for their description.
+
+
+## What installation steps from Tendrl installation documentation are not part of tendrl-ansible?
+
+This is should be clear from [Tendrl installation
+documentation](https://github.com/Tendrl/documentation/wiki/Tendrl-release-latest)
+itself, but for the sake of convenience, here is the
+list of installation or deployment steps which are out of scope of
+tendrl-ansible:
+
+* Deployment and installation of machines (either virtual or bare metal), which
+  includes setup of networking, partitioning of disks, deployment of ssh public
+  keys and so on.
+* Installation and configuration of GlusterFS on the machines, see
+  [gdeploy](https://gdeploy.readthedocs.io/en/latest/) for automation of this
+  task.
+* Setup of dedicated disk for etcd and graphite data directories.
+* Setup of https for Tendrl web and api.
+* Deployment of tls certificates and keys for etcd tls based client server
+  encryption and authentication (this means communication between various
+  tendrl components and etcd instance).
+
+
+## How do I install Tendrl with tendrl-ansbile?
+
+1)  Install tendrl-ansible:
+
+    ```
+    # yum install tendrl-ansible
+    ```
+
+    See section "How to get tendrl-ansible?" in this README file for more
+    details.
+
+2)  Create [Ansible inventory
+    file](https://docs.ansible.com/ansible/latest/intro_inventory.html) with
+    groups for `tendrl-server` and `gluster-servers`. Here is an example of
+    inventory file for 4 node cluster with Gluster:
+
+    ```
+    [gluster-servers]
+    gl1.example.com
+    gl2.example.com
+    gl3.example.com
+    gl4.example.com
+
+    [tendrl-server]
+    tendrl.example.com
+    ```
+
+3)  Add mandatory ansible variables into the inventory file you created in the
+    previous step.
+
+    This includes:
+
+    * `etcd_ip_address` configures where etcd instance is listening
+    * `etcd_fqdn` configure tendrl components to be able to connect to etcd
+      instance
+    * `graphite_fqdn` configures tendrl components (this value doesn't
+       reconfigure graphite itself!) to be able to connect to graphite instance
```

Comment:
This is awesome to now have this in the inventory file as opposed to editing them in the site.yml file!